testapps/php56_e2e/composer.json

```diff
@@ -1,6 +1,15 @@
 {
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/stanley-cheung/Protobuf-PHP"
```

Comment:
That is true. By pulling `grpc/grpc`, you are also implicitly pulling `datto/protobuf-php` (with a substituted fork) yes.